train-simulation/edge/updatePipelines.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+PROJECT=train-simulator-edge-demo
+fly -t prod set-pipeline -p ${PROJECT} -c pipeline.yaml -l ci/config.yaml  -l ci/credentials.yaml
```

Comment:
What about the PR pipeline?

Reply:
added: https://concourse.ci4rail.com/teams/edgefarm/pipelines/train-simulator-edge-demo-pull-requests